### ISSUE TITLE
Creating group doesn't work

### ISSUE THREAD

**Issue by alsomanya** [slco-2016/clientcomm#382]
Steps to replicate:
1. As a case manager user, I go to the "+ Groups" button
2. I enter my group name, select clients, and save group
Result:
I get this screen of death:
![screenshot 2017-03-14 21 40 56](https://cloud.githubusercontent.com/assets/5083453/23933941/29da38c8-08ff-11e7-99fc-aaeb7896a4c8.png)

Note: After I got this screen, I used my browser's back button and saw the group in my group list. I sent a test message to it and got a confirmation on screen, but the message never actually sent (I didn't get it on my phone). I think that the group wasn't actually created (per the error message above), even though the UI seems to indicate it was.

**Comment by tmaybe** [comment]
I can't duplicate this. I do see the behavior you describe if I don't select any clients when creating the group (and that's definitely a bug), but if I select clients, it works fine.

**Comment by alsomanya** [comment]
Oh, weird! Here's a gif of what I did — I selected a client and got that error screen... 
![mn5xt5mun0](https://cloud.githubusercontent.com/assets/5083453/23958261/c0ea9b22-095e-11e7-9708-9745456a361f.gif)

**Comment by alsomanya** [comment]
Maybe it's a problem because I'm only adding 1 client — does a group require 2+ clients to work?

**Comment by tmaybe** [comment]
That's what I was about to test :)

**Comment by tmaybe** [comment]
Yes, that's it. It works with 2+ clients, errors with 0 or 1 client.

**Comment by alsomanya** [comment]
Aha! That makes sense, and the error I got for 0 or 1 client is probably more of an edge case (my _guess_ is that the CMs who are creating groups are creating groups with multiple clients). I'll modify the bug description above and we can treat as "known-and-not-urgent".
